asminc/vic20.inc

```diff
@@ -115,3 +115,8 @@ VIA2_PCR        := VIA2+$C      ; Peripheral control register
 VIA2_IFR        := VIA2+$D      ; Interrupt flag register
 VIA2_IER        := VIA2+$E      ; Interrupt enable register
 VIA2_PA2        := VIA2+$F      ; Port register A w/o handshake
+
+; ---------------------------------------------------------------------------
+; Kernal routines, direct entries
```

Comment:
Yah, I noticed that clash in my initial go at this. :-) My first try at fixing it actually headed towards removing `PLOT` from `cbm_kernal.inc` for just the VIC-20 until that got a bit messy and I realized I wasn't entirely clear on what the broad effects of that might be. (In particular, on those not using conio and perhaps using only the assembler; do VIC-20 and C64 assembly-language developers use those files?)

So I stuck this in there for the moment, but it does seem to be going off in a somewhat different direction from how the other Kernal routine addresses are handled, and possibly should not be, given that IIRC the C64 also has its own PLOT that uses the same (but slightly displaced) routine in the C64 Kernal. I'm thinking it deserves a name there, too.

So while your suggestion that I better document this kinda-hack I did here is great, I'm wondering if you have any thoughts on what direction this ought to go in the long run, and if there's anything we can do in the short term both to improve this code and better indicate the direction for others changing things in the future. (If there's nothing obvious that the project can agree on fairly quickly, of course, we should probably just go ahead with a better comment here so the bug can get fixed and leave working out the further details for the longer term.)

Reply:
> do VIC-20 and C64 assembly-language developers use those files?

They do ;-)

Reply:
And, they expect that the entire jump table is visible, in "cbm_kernal.inc", for all Commodore models.

Your choice isn't a hack -- it's a necessity.  We would have said that you had made a hack if you had put it in "cputc.s".  But, "vic20.inc" is the best place for that model's entry-point names when they cannot be put into "cbm_kernal.inc".

I agree that the C64, also, needs the fix and the name.

Reply:
It may be a necessary hack, but it's still a hack. :-)

I've added a fixup commit for the comment; it's a separate commit for easier reviewing but I will squash it before merge. (I've marked this PR `[WIP]` to remind about this.)

I've also added another commit to extract an `UPDCRAMPTR` symbol for the C64. It's done in the same way for the same reasons, with the same comment.

Once we're finished with review of this PR I will open a new issue explaining the conio/`PLOT`/`asminc/cbm_kernal.inc` conflict problem and maybe we can think of a better solution.

Reply:
@greg-king5 So, anything more on this, or should I squash the FIXUP commit and remove the WIP?